### ISSUE TITLE
GBVisualizer (GBC) doesnt show the oscillation wave

### ISSUE THREAD

**Issue by benderscruffy** [mgba-emu/mgba#1468]
https://github.com/LIJI32/GBVisualizer

**Comment by ydnaroday** [comment]
Hello,
I would like to work on this issue, and I was just wondering if anyone could provide details to get this to work. I assume it would just be adding functionality for the registers that GBVisualizer needs, but I am not sure. Thank you!

**Comment by endrift** [comment]
Yes, you would need to add support for the registers to read the current samples. This may be more complex than it sounds. I'm not sure offhand.

**Comment by pinobatch** [comment]
Keywords for future searches: GBC PCM, PCM12 (FF76), PCM34 (FF77)
